pytube/cipher.py

```diff
@@ -27,7 +27,8 @@
 class Cipher:
     def __init__(self, js: str):
         self.transform_plan: List[str] = get_transform_plan(js)
-        var_regex = re.compile(r"^\w+\W")
+        # var_regex = re.compile(r"^\w+\W")
+        var_regex = re.compile(r"^\$*\w+\W")
```

Comment:
Unrelated to this PR

Reply:
It fixes a different issue: https://github.com/pytube/pytube/issues/1768